resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/features/DriverServiceFeatureStep.scala

```diff
@@ -54,6 +56,8 @@ private[spark] class DriverServiceFeatureStep(
     config.DRIVER_PORT.key, DEFAULT_DRIVER_PORT)
   private val driverBlockManagerPort = kubernetesConf.sparkConf.getInt(
     config.DRIVER_BLOCK_MANAGER_PORT.key, DEFAULT_BLOCKMANAGER_PORT)
+  private val  driverUIPort = kubernetesConf.sparkConf.getInt(
```

Comment:
`sparkConf.get(UI_PORT)`

Reply:
Or actually just `kubernetesConf.get(UI_PORT)`.

Reply:
Fixed this.